project/Mima.scala

```diff
@@ -32,6 +32,13 @@ object Mima {
     ProblemFilters.exclude[A]("scala.meta." + metaType)
 
   val apiCompatibilityExceptions: Seq[ProblemFilter] = Seq(
+    // Type.Placeholder
```

Comment:
it didn't pass mima :)

Reply:
ach right, I thought it's only for the class. This is a candidate for 5.x series then. For added methods minor is usually ok (though we might not have a perfect track record her), but for sure anything breaking should be 5.x